plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -89,6 +92,15 @@ const CatalogPage: FC<{}> = () => {
         hidden: location ? location?.type !== 'github' : true,
       };
     },
+    (rowData: Component) => {
+      return {
+        icon: starredEntities.has(rowData.metadata.name) ? Star : StarOutline,
+        toolTip: `${
+          starredEntities.has(rowData.metadata.name) ? 'Unstar' : 'Star'
+        } ${rowData.metadata.name}`,
+        onClick: () => toggleStarredEntity(rowData.metadata.name),
```

Comment:
Names are NOT unique identifiers for entities - the triplet of kind + namespace + name is. Or should we store their URL like we do in BIN? Because in BIN you can actually star any page, and it's by full URL.

Reply:
Since the scope of this work is for only for starring entities, I'm thinking that just using the canonnical reference of `${kind}:${namespace}:${component}` is probably best right now.

Maybe could prefix it with `entity:` to support such behaviour moving forward so theres some way to identify the different routing logic later on down the line? @freben?

Reply:
With the prefix then, sure. And if namespace is not set it's the empty string perhaps? So `entity:Component::backstage`?

Another potentially intresting question is migration of these in BIN, to the new format. :) but that's a separate story

Reply:
yeah that seem reasonable, the other thing I thought is that we could just use `default` if it didn't exist already, wasn't sure if the `catalog-backend` would return `default` as the namespace even if one wasn't provided. 

Personally, I think storing this stuff as a URI is a little less brittle, URL's can change, but URI's shouldn't.